### DISCUSSION TITLE
Py to exe

### DISCUSSION THREAD

**Discussion by Usman-Amjad** [TomSchimansky/CustomTkinter#1856] [Q&A]
I am trying to make py to exe with pyinstaller. But whenever I am trying to open that exe file the exe file opens but some widgets like entry field and treeview is not displaying in that exe kindly help me please.

**Comment by onlyforpeace** [comment]
@Usman-Amjad  is it with customtkinter?

**Reply by Usman-Amjad** [reply]
Yes, and thanks I solved the problem
That was because I was writing width, height in .place() instead of the widget parameter
